pyproject.toml

```
@@ -1,6 +1,6 @@
 [project]
 name = "yalexs-ble"
-version = "2.6.0"
+version = "3.0.0"
```

```suggestion
version = "2.6.0"
```

The release bot will take care of this